### ISSUE TITLE
Suggestion: depopulate or remove external pullups on the buttons

### ISSUE THREAD

**Issue by cibomahto** [PaxInstruments/t400-electronics#189]
External pullups are usually only necessary when the traces from button to processor input are long, or expected to be subject to large amounts of external noise. Neither of these seems to apply here, and the internal resistors should work fine for the buttons.

![screen shot 2015-08-15 at 2 46 51 pm](https://cloud.githubusercontent.com/assets/44493/9287829/9bf5c63a-435c-11e5-9adb-b470120c45bb.png)

**Comment by samchoy88** [comment]
I think these resistors keep in PCB layout but DNP in material list.

**Comment by charlespax** [comment]
Yes, Sam, let's keep them in the design as DNP. It can be useful to have them there in other derivative designs. For example, to make the LCD shield I just copied the Eagle files and removed a bunch of parts. It would be good to now have to add them back later. However, if we run into an issue we can solve by removing them, we should remove them. I made a note for production https://github.com/PaxInstruments/t400-electronics/issues/191
